modules/migration-mtc-release-notes-1-8-5.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/mtc-release-notes.adoc
+:_mod-docs-content-type: REFERENCE
+[id="migration-mtc-release-notes-1-8-5_{context}"]
+= {mtc-full} 1.8.5 release notes
+
+[id="technical-changes-1-8-5_{context}"]
+== Technical changes
+
+{mtc-first} 1.8.5 has the following technical changes:
+
+.Federal Information Processing Standard (FIPS)
+
+FIPS is a set of computer security standards developed by the United States federal government in accordance with the Federal Information Security Management Act (FISMA).
+
+Starting with version 1.8.5, {mtc-short} is designed for FIPS compliance.
+
+[id="mtc-resolved-issues-1-8-5_{context}"]
+== Resolved issues
+
```

Comment:
```suggestion
```
At some point when you're back here, you'll probably want to take out the extra line space.